src/payment/checkout/payment-form/CardHolderInformation.jsx

```diff
@@ -206,6 +215,7 @@ export class CardHolderInformationComponent extends React.Component {
               type="text"
               disabled={disabled}
               autocomplete="postal-code"
+              maxlength="10"
```

Comment:
Maybe constants?

Reply:
We don't have any current map of fields/constants. I think moving these would be added organization or abstraction for little gain right now.